R/DataBackend.R

```diff
@@ -26,7 +26,8 @@
 #'
 #' @export
 #' @examples
-#' data = data.table::data.table(id = 1:5, x = runif(5),
+#' data = data.table::data.table(
+#'   id = 1:5, x = runif(5),
```

Comment:
@mllg Is this something we want to have?

Reply:
You did not have this previously. Fix suggested in https://github.com/mlr-org/styler.mlr/pull/9/files.

Reply:
Thanks 🎉